### DISCUSSION TITLE
SEO and Google Analytic Support

### DISCUSSION THREAD

**Discussion by cdmcdermott** [adityatelange/hugo-PaperMod#412] [Q&A]
I am trying to figure out how to setup and configure google analytics and SEO

I have set the following in my config.yml file:

params:
    env: production

googleAnalytics: UA-XXXXXX-1  # Ive added my property ID here

analytics:
        google:
            SiteVerificationTag: "XYZabc"   # not sure what I need to put here ?

What else do I need to configure to get google analytics and SEO working 

Thanks

**Comment by BDHU** [comment]
One is your tracking ID, another one is your [verification tag](https://support.google.com/analytics/answer/1008080?hl=en). As long as your website is verified the verification tag doesn't really matter

**Reply by cdmcdermott** [reply]
ok perfect I think the analytics is now working. How does the SEO work? Once we have set the env to production what else do we need to do ?

**Reply by BDHU** [reply]
Generally, you should be good to go. Make sure you actually inform the search engine about the new URLs by, e.g. submitting a sitemap to webmaster. You are probably better off searching for how to improve SEO for a website as that is a much larger subject to cover. Examples like including proper tags, descriptions, etc. There are plenty of examples online both covering Hugo specifics and more general stuff regarding SEO.

**Comment by benedicttwl** [comment]
Submitting a follow-up question, can I set it up with GA4? 

In config.toml, 

`[params.analytics.google]`
SiteVerificationTag = "G-XXXXXXXX"

The Analytics support pages said to add the Global site tag (gtag.js) script into the **head** section of the HTML. 

I added this to the `public > public > index.html` file. But I have no idea if I'm doing it right.

**Reply by aormsby** [reply]
Did this ever work for you? I'm having trouble figuring this out as well.

**Reply by aormsby** [reply]
Never mind, I have it. I used the `G-xxxxx` tag for both of the params.

```yaml
googleAnalytics: G-xxxxx


analytics:
    google:
        SiteVerificationTag: "G-xxxxx"
```

Don't know if that's necessary, but it works! Posted this for others in case they look here.
